packages/zero-solid/src/solid-view.ts

```diff
@@ -26,17 +20,16 @@ export type QueryResultDetails = {
   readonly type: ResultType;
 };
 
-type State = [Entry, QueryResultDetails];
+export type State = [Entry, QueryResultDetails];
 
-const complete = {type: 'complete'} as const;
-const unknown = {type: 'unknown'} as const;
+export const COMPLETE: QueryResultDetails = {type: 'complete'} as const;
```

Comment:
Maybe we should freeze these to prevent more surprises

Reply:
done.  this actually modifies solid's behavior as it checks for frozen objects, but i think its best not to rely on it.